code/game/objects/items/weapons/tools/crowbar_vr.dm

```diff
@@ -4,23 +4,30 @@
 
 /obj/item/weapon/tool/prybar
 	name = "pry bar"
-	desc = "A steel bar with a wedge. It comes in a variety of configurations - collect them all."
+	desc = "A steel bar with a wedge, designed specifically for opening unpowered doors in an emergency. It comes in a variety of configurations - collect them all!"
 	icon = 'icons/obj/tools_vr.dmi'
 	icon_state = "prybar"
 	item_state = "crowbar"
-	flags = CONDUCT
-	slot_flags = SLOT_BELT
 	force = 4
-	throwforce = 6
-	pry = 1
-	w_class = ITEMSIZE_SMALL
-	origin_tech = list(TECH_ENGINEERING = 1)
-	matter = list(DEFAULT_WALL_MATERIAL = 30)
-	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
+	throwforce = 1
+	matter = list(DEFAULT_WALL_MATERIAL = 25)
+	attack_verb = list("whapped", "smacked", "swatted", "thwacked", "hit")
 	usesound = 'sound/items/crowbar.ogg'
 	toolspeed = 1
 	var/random_color = TRUE
 
+// Todo: Prevent it from prying up floor boards. Only affect doors. I couldn't remember how to do this. -Ace
+/*
+/obj/item/weapon/tool/prybar/on_hit()
+	if(istype(/obj/machinery/door/airlock))
+		..()
+	else if(istype(/mob)
+		..()
+	else
+		// Tell the player that they can't use this on anything except airlocks.
```

Comment:
@Arokha or @Cameron653 can probably figure out how to do this bit. I'm so rusty and they appear to be offline at the moment.